src/commcare_cloud/ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -73,6 +73,12 @@ effective_cache_size = {{ postgresql_effective_cache_size }}
 enable_nestloop = {{ enable_nestloop }}
 {% endif %}
 
+# PARALLEL QUERY
+{% if postgresql_max_parallel_degree > 0 %}
```

Comment:
the only thing that must be done in order to take advantage of parallel query is to set max_parallel_degree to a value greater than zero. A value between 1 and 4 is recommended. The number should be set with due regard for max_worker_processes, which limits the number of background worker processes that may exist at any one time. This value includes not only parallel workers but any other background worker processes which may have been configured
https://wiki.postgresql.org/wiki/Parallel_Query